prime-router/src/testIntegration/resources/datatests/translation-test-config.csv

```diff
@@ -8,6 +8,7 @@ HL7_to_INTERNAL/CA-20211001-sully.hl7,hl7/test-covid-19,HL7_to_INTERNAL/CA-20211
 HL7_to_INTERNAL/LP-covid-19-20220124002222.hl7,hl7/lifepoint-covid-19,HL7_to_INTERNAL/LP-covid-19-20220124002222.internal.csv,covid-19,INTERNAL,PASS,"","ignore.ignore-lifepoint","","",""
 HL7_to_INTERNAL/monkeypox-20221110.hl7,monkeypox,HL7_to_INTERNAL/monkeypox-20221110.internal.csv,monkeypox,INTERNAL,PASS,"current_date","ignore.ignore-monkeypox","","",""
 HL7_to_FHIR/sample_orm_20230809-001.hl7,,HL7_to_FHIR/sample_orm_20230809-001.fhir,,FHIR,PASS,"","","","",""
+HL7_to_FHIR/xpn_fn_to_humanname_mapping_hl7_src.hl7,,HL7_to_FHIR/xpn_fn_to_humanname_mapping_fhir_dest.fhir,,FHIR,PASS,"","","","",""
```

Comment:
The mapping inventory work has been using a different integration test suite that does a full HL7v2 to FHIR to HL7v2 test using only the path to the test files as input.

Reply:
yea, I saw that, and these tests are added solely to check the FN mapping changes in the context of Patient (fn-xpn-Patient) and Practitioner (fn-xcn-Practitioner) without doing XPN, XCN mapping, we can disregard these tests and go with tests furnished with those mapping PRs actually merged.

Also a question: does this mean that at the end of the mapping implementation compaign, the tests using translation-test-config.csv will be deprecated?

Reply:
"Also a question: does this mean that at the end of the mapping implementation compaign, the tests using translation-test-config.csv will be deprecated?"

probably no, there are sender transform tested ....